lang/java/reef-common/src/main/java/org/apache/reef/driver/evaluator/EvaluatorRequest.java

```diff
@@ -196,6 +209,12 @@ private Builder(final EvaluatorRequest request) {
       }
     }
 
+    @SuppressWarnings("checkstyle:hiddenfield")
```

Comment:
Every existed setters in `EvaluatorRequest.Builder` takes a parameter that has the same name as its local variable like below.
```java
public T setMemory(final int megaBytes) {
      this.megaBytes = megaBytes;
      return (T) this;
}
```
IMHO, previous contributor seems to have turned this check off, because it's not really that confusing. So I decided to follow these precedents.